tests/test_renderers/fixtures/syntax_elements.md

```diff
@@ -139,6 +139,18 @@ foo
         foo
 .
 
+--------------------------
+Fenced Code no language with trailing whitespace:
+.
+```  
```

Comment:
this is the exact same as the test before, but w/ two spaces after the first fence